deploy/ansible/run.sh

```diff
@@ -16,20 +16,22 @@ wireguard_server_private_key=$WIREGUARD_SERVER_PRIVKEY
 elkElasticUser=$ELASTIC_USER
 elkElasticPassword=$ELASTIC_PASSWORD
 grafanaAdminPassword=$GRAFANA_ADMIN_PASSWORD
+
 NODE_01_ACCOUNTID=$NODE_01_ACCOUNTID
-NODE_01_VALIDATOR_PRIVKEY=$NODE_01_VALIDATOR_PRIVKEY
-NODE_01_P2PIDENTITYPRIVATEKEY=$NODE_01_P2PIDENTITYPRIVATEKEY
 NODE_02_ACCOUNTID=$NODE_02_ACCOUNTID
-NODE_02_VALIDATOR_PRIVKEY=$NODE_02_VALIDATOR_PRIVKEY
-NODE_02_P2PIDENTITYPRIVATEKEY=$NODE_02_P2PIDENTITYPRIVATEKEY
 NODE_03_ACCOUNTID=$NODE_03_ACCOUNTID
+
+NODE_01_VALIDATOR_PRIVKEY=$NODE_01_VALIDATOR_PRIVKEY
```

Comment:
Do not rename environment variables on the right side and use a consistent scheme across all of them.

Reply:
Order them by node too